services/clusters-keeper/src/simcore_service_clusters_keeper/data/prometheus.yml

```diff
@@ -0,0 +1,51 @@
+global:
+  scrape_interval: "30s"
+scrape_configs:
+  # Make Prometheus scrape itself for metrics.
```

Comment:
probably this isnt strictly necessary to have here, but fine

Reply:
true, will remove